backends-velox/benchmark/ColumnarTableCacheBenchmark-results.txt

```diff
@@ -0,0 +1,23 @@
+OpenJDK 64-Bit Server VM 1.8.0_322-b06 on Mac OS X 13.5
+Apple M1 Pro
+table cache count:                        Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+disable columnar table cache                      16773          17024         401          1.2         838.7       1.0X
+enable columnar table cache                        9985          10051          65          2.0         499.3       1.0X
+
+
+OpenJDK 64-Bit Server VM 1.8.0_322-b06 on Mac OS X 13.5
+Apple M1 Pro
+table cache column pruning:               Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+disable columnar table cache                      16429          16873         688          1.2         821.5       1.0X
+enable columnar table cache                       15118          15495         456          1.3         755.9       1.0X
```

Comment:
the perfomance of column pruning has less benefit due to now we always deserialize the whole columns for columanr table cache.